how-to/customize-workspace/docs/how-to-add-integrations-to-home.md

```diff
@@ -164,7 +164,8 @@ public async getHelpSearchEntries?(): Promise<HomeSearchResult[]> {
             icon: "http://localhost:8080/favicon.ico,
             actions: [],
             data: {
-                providerId: "my-integration"
+                providerId: "my-integration",
+                populateQuery: "/open-site"
```

Comment:
Shouldn't this query be set by the module instead of via data? i.e. if they update the version of the module they use they will need to remember to check to see if the populate query has changed.

Reply:
That code is in the module, it returned by the getHelpSearchEntries